### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by Lawmsangi**
On this project: 
- I used ES6 modules to write modular JavaScript.
- I used proper ES6 syntax.
- I used Luxon library to get Time and Date.

### CODE REVIEW

File: modules/ShowNavigation.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export const showBooksSec = document.getElementById('book-list-section');
+export const addBooksSec = document.getElementById('add-book-section');
+export const contactSec = document.getElementById('contact-section');
+export const list = document.getElementById('show-list');
+// eslint-disable-next-line import/prefer-default-export
+export const add = document.getElementById('show-add');
```
Comment (shella12) [inline modules/ShowNavigation.js]: - [ ] Linters help us maintain consistent coding standards and avoid potential errors. Please avoid using inline comments to disable them

File: modules/Date.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
+import { DateTime } from './Luxon.js';
+
+export const displayTime = () => {
+  const now = DateTime.now();
+  const date = document.getElementById('date');
+  date.innerHTML = now.toLocaleString(DateTime.DATETIME_FULL);
+};
```
Comment (shella12) [inline modules/Date.js]: - [ ] Displaying the seconds along with hrs and minutes and updating the dateTime automatically can enhance the user experience and accuracy of your app. Please add these features to your date display and ensure that it refreshes dynamically without requiring a page reload. See [Get the current date and time](https://github.com/moment/luxon/blob/master/docs/tour.md#get-the-current-date-and-time).

File: modules/ClassBook.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line import/prefer-default-export
+export class Book {
```
Comment (shella12) [inline modules/ClassBook.js]: - [ ] As mentioned before please avoid using inline comments to disable  linters as they help us maintain consistent coding standards and avoid potential errors. Try to resolve the error manually by using the alternate solution.

File: modules/LocalStorage.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line import/prefer-default-export
+export const storeBooks = (books) => {
```
Comment (shella12) [inline modules/LocalStorage.js]: - [ ] Here too please don't disable linters.

File: modules/RemoveBook.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable */
+import { getBooks, storeBooks } from './LocalStorage.js';
```
Comment (shella12) [inline modules/RemoveBook.js]: - [ ] And here as well please remove the disable linters comments

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @Lawmsangi, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- There is a modules directory with other .js files ✅ 
- `index` files  are added ✅   
- used `arrow` functions ✅ 
- imported `luxon` ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alusp — APPROVED** [review]
Hi @Lawmsangi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations

## Highlights

- [ ] No Linters Errors :heavy_check_mark: 
- [ ] Professional PR Summary :heavy_check_mark: 
- [ ] Use arrow functions :heavy_check_mark: 
- [ ] imported luxon :heavy_check_mark: 
- [ ] Use modules approach :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**